components/src/CartCreate/cart.create.tsx

```diff
@@ -0,0 +1,357 @@
+        .catch((error) => {
+          // eslint-disable-next-line no-console
+          console.error(error.message);
+        });
+    });
+  }
+
+  handleChange(event) {
+    this.setState({ cartName: event.target.value });
+  }
+
+  handleAddNewCart() {
+    const { cartName } = this.state;
+    if (cartName.length > 0) {
+      this.createNewCart();
+    }
+  }
+
+  clearCartNameField() {
+    this.setState({ cartName: '' });
+  }
+
+  handleCartFieldChange(event, index) {
+    const { cartElements } = this.state;
+    const elements = [...cartElements];
+    elements[index] = { ...elements[index] };
+    elements[index].cartName = event.target.value;
+    this.setState({ cartElements: elements });
+  }
+
+  clearCartNameFieldItem(index) {
+    const { cartElements } = this.state;
+    const elements = [...cartElements];
+    elements[index] = { ...elements[index] };
+    elements[index].cartName = '';
+    this.setState({ cartElements: elements });
+  }
+
+  handleCartRename(index) {
+    const { cartElements } = this.state;
+    const { handleCartsUpdate } = this.props;
+    if (cartElements[index].cartName.length > 0) {
+      const cartElem = [...cartElements];
+      cartElem[index] = { ...cartElem[index] };
+      cartElem[index].showLoader = true;
+      this.setState({ cartElements: cartElem });
+      login().then(() => {
+        cortexFetch(`/${cartElements[index]._descriptor[0].self.uri}`, {
+          method: 'put',
+          headers: {
+            'Content-Type': 'application/json',
+            Authorization: localStorage.getItem(`${Config.cortexApi.scope}_oAuthToken`),
+          },
+          body: JSON.stringify({ name: cartElements[index].cartName }),
+        })
+          .then(() => {
+            this.fetchCartData();
+            handleCartsUpdate();
+            const elements = [...cartElements];
+            elements[index] = { ...elements[index] };
+            elements[index].editMode = false;
+            elements[index].showLoader = false;
+            this.setState({ cartElements: elements });
+          })
+          .catch((error) => {
+            // eslint-disable-next-line no-console
+            console.error(error.message);
+          });
+      });
+    }
+  }
+
+  handleCartSelect(el, index) {
+    const { handleCartElementSelect, handleModalClose } = this.props;
+    this.setState({ selectedElement: index });
+    handleCartElementSelect(index);
+    handleModalClose();
+  }
+
+  renderCartItems() {
+    const { cartElements, selectedElement } = this.state;
+    if (cartElements.length) {
+      return cartElements.map((el, index) => (
+        <li className={`carts-list-item ${selectedElement === index ? 'selected' : ''} ${el.editMode ? 'edit-mode-state' : ''}`} key={`cartItem_${el._descriptor[0].name ? el._descriptor[0].name.trim() : 'default'}`} role="presentation" onClick={() => this.handleCartSelect(el, index)}>
+          <h4 className="cart-info">{el._descriptor[0].name || 'Default'}</h4>
```

Comment:
Instead of hardcoding `Default`, we should use the string property from the locale files